src/open_samus_returns_rando/files/custom_scenario.lua

```diff
@@ -54,3 +54,53 @@ function Scenario.UpdateDNACounter()
   
   GUILib.UpdateDNACounter(currentDNA, maxDNA)
 end
+
+Scenario.IngameScenarios = {
+  "s000_surface",
+  "s010_area1",
+  "s020_area2",
+  "s025_area2b",
+  "s028_area2c",
+  "s030_area3",
+  "s033_area3b",
+  "s036_area3c",
+  "s040_area4",
+  "s050_area5",
+  "s060_area6",
+  "s065_area6b",
+  "s067_area6c",
+  "s070_area7",
+  "s090_area9",
+  "s100_area10",
+  "s110_surfaceb"
+}
+
+local original_init = Scenario.InitScenario
+function Scenario.InitScenario(_ARG_0_, _ARG_1_, _ARG_2_, _ARG_3_)
+  original_init(_ARG_0_, _ARG_1_, _ARG_2_, _ARG_3_)
+
+  for k ,v in ipairs(Scenario.IngameScenarios) do
+    if Scenario.CurrentScenarioID == v then
```

Comment:
Any better idea how to find out if we are in a ingame scenario? Otherwise it tried to call it for the cockpit scenario. oof!

Reply:
Check for c10_samus?

Reply:
👍